drivers/mpu9250/MPU9250.cpp

```diff
@@ -160,8 +160,15 @@ int MPU9250::mpu9250_init()
 	//	DF_LOG_ERR("sample rate config failed");
 	//}
 	//usleep(1000);
+
+#if defined(__EDISON)
+	//FIFO corrupt and overflow at 250Hz.
+	result = _writeReg(MPUREG_CONFIG,
+			   BITS_DLPF_CFG_184HZ | BITS_CONFIG_FIFO_MODE_OVERWRITE);
```

Comment:
Don't you have the same problem on your device? The temperature sensor samples at 4000Hz (using DLPF_CFG = 0) and it is faster then frequency of _measure, or i don't understand something?